pkgs/ffigen/lib/src/logger.dart

```diff
@@ -7,14 +7,16 @@ import 'dart:io';
 import 'package:logging/logging.dart';
 
 /// Creates a default logger that logs to stdout and stderr.
-Logger createDefaultLogger() {
+Logger createDefaultLogger([Level level = Level.INFO]) {
```

Comment:
Consider aligning with the other packages:

```dart
Logger _createTestLogger({
  List<String>? capturedMessages,
  Level level = Level.ALL,
}) => Logger.detached('')
  ..level = level
  ..onRecord.listen((record) {
    printOnFailure('${record.level.name}: ${record.time}: ${record.message}');
    capturedMessages?.add(record.message);
  });
```

`printOnFailure` is great for running the tests without any output when there are no failures.

(IHMO there should be 0 output except for the succeeds test number when running `dart test` in the packages.)